source/index.html

```diff
@@ -1,12 +1,157 @@
+              <p class="advantages__description">Замените один-два приема
+                обычной еды на наше
+                функциональное питание.</p>
+            </li>
+            <li class="advantages__item">
+              <img class="advantages__image" src="img/time-4.png">
+              <p class="advantages__description">Уже через месяц наслаждайтесь
+                изменениями к лучшему
+                вашего питомца!</p>
+            </li>
+          </ul>
+        </section>
+        <section class="before-after">
+          <div class="before-after__wrapper container">
+            <div class="before-after__content" >
+              <h2 class ="before-after__title">Живой пример</h2>
+              <p class="before-after__description">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день. </p>
+              <div class="before-after__result">
+                <dl class="before-after__list">
+                  <div class="was-became__item before-after-item">
+                    <dt class =" before-after-item__value">5кг</dt>
+                    <dd class ="before-after-item__title">снижение веса</dd>
+                  </div>
+                  <div class="before-after__item before-after-item">
+                    <dt class ="before-after-item__value">60 дней</dt>
+                    <dd class ="before-after-item__title">затраченного времени</dd>
+                  </div>
+                </dl>
+                <p class="before-after__price">затраты на питание:
+                  <span class="was-became__price-value">15000 руб.</span>
+                </p>
+              </div>
+              <div class="slider">
+                <div class="slider__img">
+                  <img src="" alt="Кот до корма">
+                  <img src="" alt="Кот после корма">
+              </div>
+              <div class="slider__bar">
+                <button class="slider__text slider__text--before" type="button">Было</button>
+                <div class="slider__range">
+                  <button class="slider__button" type="button" aria-label="лево право"></button>
+                </div>
+                <button class="slider__text slider__text--after" type="button">Стало</button>
+              </div>
+            </div>
+          </div>
+          </div>
+        </section>
+        <section class="contacts">
+          <h2 class="visually-hidden">Контакты</h2>
+          <div class="contacts__wrapper">
+            <div class="contacts__address address">
+              <div class="address__container container">
+                <b class="address__title">Приглашаем к сотрудничеству дилеров!</b>
+                <p class="address__text"><span class="address__text-wrap">ул. Большая Конюшенная, д. 19/8</span> Санкт-Петербург</p>
+              </div>
```

Comment:
Можно использовать более семантические тэги
```suggestion
              <address class="address__container container">
                <strong class="address__title">Приглашаем к сотрудничеству дилеров!</strong >
                <p class="address__text"><span class="address__text-wrap">ул. Большая Конюшенная, д. 19/8</span> Санкт-Петербург</p>
              </address>
```